robotiq_c_model_control/nodes/autodetect_gripper_node.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+"""
+Copyright 2018 by READY Robotics Corporation.
```

Comment:
I think the general practice is to follow the oldest Copyright date.

Reply:
I might have asked the wrong question here. I was more or less wondering, since when I created new files in my tutorial I was asked to make them say "2020" why, when there are new files in a different repository, we don't update them. But, judging based on context, it seems these are not actually new files, so that would make sense why you would keep the older date.